spark/processing/3.5/py3/docker/py39/Dockerfile.cpu

```diff
@@ -0,0 +1,142 @@
+FROM public.ecr.aws/amazonlinux/amazonlinux:2023
```

Comment:
is this the base image recommended? worry is that there is 2023 suffix in the end.

Reply:
EMR 7.0.0 uses AL2023, we have to use this